wgpu-hal/src/gles/command.rs

```diff
@@ -489,10 +503,12 @@ impl crate::CommandEncoder<super::Api> for super::CommandEncoder {
                     }
                 }
 
-                // set the draw buffers and states
-                self.cmd_buffer
-                    .commands
-                    .push(C::SetDrawColorBuffers(desc.color_attachments.len() as u8));
+                if !rendering_to_external_framebuffer {
+                    // set the draw buffers and states
+                    self.cmd_buffer
+                        .commands
+                        .push(C::SetDrawColorBuffers(desc.color_attachments.len() as u8));
```

Comment:
How does the framebuffer get set when rendering to an external framebuffer?

Reply:
That happens down in `set_attachment`: https://github.com/gfx-rs/wgpu/pull/2609/files#diff-c91d45156b742345a9fe20a6742d5fbc22e2c546a80291a227b1f3bf852052e3R131 which is called by the `C::BindAttachment` command I believe